src/test/java/POM/BasePage.java

```diff
@@ -11,11 +11,11 @@
 import java.time.Duration;
 
 public class BasePage {
-    WebDriver driver;
+   protected WebDriver driver;
     WebDriverWait wait;
```

Comment:
Can be removed `WebDriverWait wait;`because it is not used anywhere in your code